$NetBSD$

1) Install manpages into $PREFIX/man instead of $PREFIX/share/man

--- install.orig	2019-01-19 05:55:40.000000000 +0000
+++ install
@@ -103,9 +103,9 @@ Install()
 	rm -r lib/obnc/obncdoc
 
 	#install man pages
-	EchoAndRun mkdir -p "$destdir$prefix/share/man/man1"
+	EchoAndRun mkdir -p "$destdir$prefix/man/man1"
 	for file in $man1Files; do
-		EchoAndRun cp "share/man/man1/$file" "$destdir$prefix/share/man/man1"
+		EchoAndRun cp "share/man/man1/$file" "$destdir$prefix/man/man1"
 	done
 
 	#install obncdoc style file
